docs/data/date-pickers/validation/validation.md

```diff
@@ -13,7 +13,7 @@ All the date and time pickers have an API for adding validation constraints.
 By default, they provide visual feedback if the component value doesn't meet the validation criteria.
 
 :::info
-The validation props are showcased for each type of picker component using the responsive pickers (`DatePicker`, `TimePicker`, `DateTimePicker`, and `DateRangePicker`)
+The validation props are showcased for each type of picker component using the responsive pickers (`DatePicker`, `TimePicker`, `DateTimePicker`, `DateRangePicker` and `DateCalendar`)
```

Comment:
`DateCalendar` is not a responsive picker component. 🤔 
Technically, validation props are not showcased for `DateCalendar` and `TimeClock`, it's just one of instance showing how it looks on those components, but everything else is listed only on `Pickers` level components. 👌 
```suggestion
The validation props are showcased for each type of picker component using the responsive pickers (`DatePicker`, `TimePicker`, `DateTimePicker`, and `DateRangePicker`)
```